docs/intro/ff-ui/resource-hierarchy.md

```diff
@@ -0,0 +1,34 @@
+---
+slug: resource-hierarchy
+title: Resource Hierarchy Overview
+tags: []
+description: Explore the Resource Hierarchy Overview to understand the correlation between traditional Flutter app components and their equivalents in FlutterFlow.
+---
+
+# Resource hierarchy overview
```

Comment:
I'm a little unsure about this page and it feels like it is directed at Flutter developers more, but anyone else coming to FlutterFlow will be confused about it. I understand this was there in Leigha's talk but that was talk was also for Flutter devs so I'm not sure if this fits completely here. We can pick and choose and just talk about the components of a FlutterFlow project/app and not have the comparison with Flutter maybe? 

@leigha your thoughts here too please, thanks!

Reply:
So the purpose of this page is to just give a brief overview of how Flutter components are translated into FF and then direct users to dedicated pages in our doc to understand more. I agree that it is more targeted towards the Flutter developers, so maybe we can change its location. @leighajarett wdyt?

Reply:
Maybe we do the same diagram but without the comparison in this particular page and I'm assuming resource hierarchy would give an understanding of all the resource categories we have under Resources folder. So like Projects, Pages, Widgets etc. So this should be unrelated to Flutter. But when we talk about generated code, we can add a comparison there. (it will come later) And it will mostly be a FlutterFlow to Flutter code comparison rather than Flutter to FlutterFlow. Would that make sense?

Reply:
I think if we remove the comparison, keeping the diagram doesn't make sense. Instead, I think we could add a 'Resources' tile on the homepage that redirects users to the resources folder. Also, this page is crucial for anyone transitioning from Flutter, we could move it to a more appropriate location.

Reply:
Sorry I meant that we can have a diagram only for the resources hierarchy of FlutterFlow project, so just removing the Flutter terms from the diagram and keeping the FlutterFlow terms. 

My suggestion to cater to both Flutter + FlutterFlow is to do the following flow: 

1. Show diagram with just FlutterFlow terms and 2 line explanations of the resource hierarchy -- Projects, Page, etc. 
2. Next heading section "Flutter to FlutterFlow" talks about "if you are coming from Flutter...then.." And this section shows the original diagram and the comparisons. 
Wdyt?

Reply:
Got it. Updated accordingly. Thanks.